Client/Frontend/Browser/Tabs/TabTrayViewController.swift

```diff
@@ -101,12 +177,34 @@ class TabTrayViewController: UIViewController, TabTrayController {
         return segmentedControl
     }
 
+    private func createButtonItem(imageName: String,
+                                  action: Selector,
+                                  a11yId: String,
+                                  a11yLabel: String) -> UIBarButtonItem {
+        let button = UIBarButtonItem(image: UIImage.templateImageNamed(imageName),
+                                     style: .plain,
+                                     target: self,
+                                     action: action)
+        button.accessibilityIdentifier = a11yId
+        button.accessibilityLabel = a11yLabel
+        return button
+    }
+
     @objc
     private func segmentChanged() {
         segmentPanelChange()
     }
 
     private func segmentPanelChange() {}
+
+    @objc
+    func deleteTabsButtonTapped() {}
+
+    @objc
+    func newTabButtonTapped() {}
+
+    @objc
+    func doneButtonTapped() {}
 }
 
 extension TabTrayViewController: UIToolbarDelegate {}
```

Comment:
Just noticed, empty extension. Should UIToolbarDelegate comformance be applied on the class definition?